ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java

```diff
@@ -434,6 +440,14 @@ public void setIsHiveServerQuery(boolean isHiveServerQuery) {
     this.isHiveServerQuery = isHiveServerQuery;
   }
 
+  public boolean isCompactionSession() {
```

Comment:
minor: could we rename to simple `isCompaction()`?

Reply:
done